academy/ibc/what-is-ibc.md

```diff
@@ -0,0 +1,84 @@
+
+![Internet of blockchains](images/internetofchains.png)
+
+Although application-specific blockchains offer superior scalability compared to general-purpose blockchain platforms, smart contract development for general-purpose chains and generic virtual machines (VMs), e.g. Ethereum, offer their own benefits. IBC provides a method of incorporating the strengths of general-purpose and application-specific blockchains into unified overall designs. It allows, for example, a Cosmos chain tailored towards performance and scaleability to use funds that originate on Ethereum and possibly record events in a Corda distributed ledger or, in the reverse, a Corda ledger initiating the transfer of underlying assets defined in Cosmos or Ethereum.
+
+New concepts such as Interchain Accounts will continue to further increase adoption times and the possibilities IBC offers: a blockchain will be able to control an account on another blockchain with Interchain Accounts.
+
+## Architecture
+
+Assume we have two chains: **chain *A* and chain *B***. Both will implement the ICS for IBC.
+
+![IBC overview](images/ibcoverview.png)
+
+Notice three crucial elements in the diagram above:
+
+* The chains use relayers to communicate;
+* Chains can use different relayers to send messages to each other;
+* Each side of the relay uses the light client of the other chain to quickly verify incoming messages.
+
+In fact, the relayer listens and picks up messages from a channel to forward them. Both chains implement IBC and first initiate a connection to then establish channels. Notice that relayers ensure that both chains know that a package has been transmitted.
+
+![IBC module](images/ibcmodule.png)
+
+The interface exposed by the standard IBC implementation is referred to as the [IBC handler](https://github.com/cosmos/ibc/tree/master/spec/core/ics-025-handler-interface). Modules can use the IBC handler for client, connection, and channel lifecycle management, and to send and receive packets.
+
+![IBC handler](images/ibchandler.png)
+
+For each connection between two blockchains, different channels can be established. **Channels** provide the ordered exactly-once delivery of packets and handle module permissioning through **ports**. A **module** is bound to a port and channels are end-to-end between two ports. The relevant abstraction layer for IBC application developers is that of channels and ports. IBC modules can bind to any number of ports.
```

Comment:
@charleenfei The comment about channel and ports being the relevant abstraction layers I recognized from here: https://ibc.cosmos.network/main/ibc/overview.html#what-is-the-interblockchain-communication-protocol-ibc

So essentially: "To have your module interact over IBC you must: bind to a port(s), define your own packet data and acknolwedgement structs as well as how to encode/decode them, and implement the IBCModule interface."

This would make sense after having made the distinction IBC TAO vs IBC APP